bin/sentieon-genomics-202010.02/lib/python/sentieon/vcflib/tribble.py

```
@@ -0,0 +1,320 @@
+# Copyright (c) 2014-2021 Sentieon Inc. All rights reserved
+import bisect
+import collections
+import heapq
+import io
+import os
+import struct
+import time
+
+from .compat import *
```

Ruff (0.8.2)

`from .compat import *` used; unable to detect undefined names

(F403)